### PR TITLE
Login moved from Django to Angular

### PR DESCRIPTION
**Pull request by weronikafelczak**
__Description:__

#689

__New imports / dependencies:__

* `package`

__Unit Tests:__

* `apps/volontulo/tests/views/test_auth.py`

__What tests do I need to run to validate this change:__

Description of testing workflow and potential affected places.

### CODE REVIEW

**Comment by magul** [comment]
@weronikafelczak generally speaking You need to:
 * Create proper `/login` route in Angular routing (see: https://github.com/CodeForPoznan/volontulo/blob/master/frontend/src/app/app.module.ts#L19)
 * Attach this route with newly created Angular component,
 * Create template for this component and most probably some css file
 * This template need to contain login form made in Angular
 * Business logic of this form need to be stored in mentioned above component
 * This component need to use AuthService as authorization provider (see: https://github.com/CodeForPoznan/volontulo/blob/master/frontend/src/app/auth.service.ts)
 * After creation of this route/component/tempalte You need to move all links to login from Django to Angular (see `ANGULAR_ROOT` in Django settings and `djangoRoot` in Angular)
 * Old login form need to be removed from login view or login view need to be removed at all from Django (if register form is already moved to Angular)
 * Old route for Django login (`/o/login`) need to be removed.

There a lot to do, but if You have done Angular tutorial, it's definetly in your reach. @Bosmanfrx or @radoslawroszkowiak will help You on todays meeting.

**Comment by magul** [comment]
@weronikafelczak for unknown reason You have `package.json.lock` in this change. We don't use yarn, we use pretty old npm so it should not be generated.

I would say, that You use yarn or other npm not from container, but from your host environment.

You can ask @Bosmanfrx or @TrebuhD how to install dependencies inside docker container.

**Comment by magul** [comment]
@weronikafelczak and You have multiple errors on Travis, see: https://travis-ci.org/CodeForPoznan/volontulo/builds/286648343

File: frontend/src/app/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { NgModel } from '@angular/forms';
+import { AuthService } from '../auth.service';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
```
[the remark this answers is not in the document]
Comment (weronikafelczak) [reply frontend/src/app/login/login.component.ts]: I guess Component and AuthService have to stay?

**Pull request merged**